### PR TITLE
Connecting

### PR DESCRIPTION
**Pull request by AlexDresco**
Barcode should go from Manual.js to Results.js (through the screens)

### CODE REVIEW

File: screens/ResultsScreen.js [hunk 1]
```diff
@@ -5,31 +5,18 @@ import Results from '../components/Results'
...
-            index: 0,
-            actions: [
-              NavigationActions.navigate({ routeName: 'HomeScreen' })
-            ],
-          }))
-        }}
+        onPress={() => this.props.navigation.navigate('HomeScreen')}
         />
       </View>
     );
```
Comment (Din-s1) [inline screens/ResultsScreen.js]: So few lines 👍 its beautiful 🥇 💯

File: components/InputScanner.js [hunk 2]
```diff
@@ -33,12 +32,12 @@ class InputScanner extends React.Component {
...
           <Text>Requesting for camera permission</Text> :
-          this.state.hasCameraPermission === false ?
-            <Text>Camera permission is not granted</Text> :
-            <BarCodeScanner
+        this.state.hasCameraPermission === false ?
+          <Text>Camera permission is not granted</Text> :
+          <BarCodeScanner
               onBarCodeRead={this._handleBarCodeRead}
               style={{ height: 200, width: 200 }}
-            />
```
Comment (Din-s1) [inline components/InputScanner.js]: Why do these have weird block next to them in code review?

File: App.js [hunk 3]
```diff
@@ -5,19 +5,18 @@ import { createStackNavigator, StackActions, NavigationActions } from 'react-nav
...
 
 
-export default createStackNavigator({
-  HomeScreen: {
-    screen: HomeScreen,
-  },
-  InputScreen: {
-    screen: InputScreen,
+
+
```
Comment (Din-s1) [inline App.js]: Extra lines

File: components/InputScanner.js (hunk 2 above)
Comment (AlexDresco) [reply components/InputScanner.js]: @Din-s1 The ones which if you click on it they expand? They're just code that hasn't been changed and is hidden for space :)

File: App.js (hunk 3 above)
Comment (AlexDresco) [reply App.js]: Yeap need to clean up! I tried to look for packages on atom to: 

- Remove extra lines
- Remove extra import not used in file

Definitely to add on Trellololo.

**Pull request merged**